phpat

```diff
@@ -18,6 +18,10 @@ if (PHP_VERSION_ID < 70100) {
     exit(1);
 }
 
+error_reporting(E_ALL);
+ini_set('display_errors', 'stderr'); // send php warnings to stderr
```

Comment:
Could we decide it depending on the dry-run option? It should be easy with something like:
```
$options = $input->getOptions();
$stream = isset($options['dry-run']) ? 'stdout' : 'stderr';
```

Reply:
Not sure why you want that.. could you elaborate?

Content printed on stderr are visible on the regular console (in case that was your idea)

Reply:
(Having  errors on stderr is one of the important points from https://medium.com/@jdxcode/12-factor-cli-apps-dd3c227a0e46)

Reply:
It is a good practice and it has to be the default stream for errors.
I just want to provide a dry-run option on the tool. The idea of this optional flag is to report errors on stdout and exit without an error code, so for instance internal errors don't cut the flow on CI tools.

Reply:
IMO the actual output of the tool will always be sent on stdout.

stderr is meant for error messages only

Reply:
(With this line we do not redirect any output to stderr.. but just ones triggered by native php and the error- handler mechanics)

Reply:
Currently I'm sending the validation errors to stderr and you are right, this is not the place.
Regarding php errors, I still think this should be something that could be optional to the user, but I will investigate a bit more how other tools do it and decide based on that.
Thanks!

Reply:
In case you merge the PR only with the dry-run option I will adjust the sources happily